symfony/twig-bundle/3.3/templates/base.html.twig

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
```

Comment:
Can we please change this line to:

```html
<meta charset="UTF-8">
```

Notes:

* I've validated the current code using https://validator.w3.org/#validate_by_input and putting `/>` in `<meta>` is valid.
* However, the HTML5 spec never includes it: https://www.w3.org/TR/html5/document-metadata.html#the-meta-element
* MDN doesn't include it either: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta

So, even if it could be valid for BC purposes, it looks like `<meta ... />` should be `<meta ...>` (same for other empty elements).

Reply:
The `/` is valid and mentioned in the spec:
https://www.w3.org/TR/html5/syntax.html#start-tags
> If the element is one of the void elements, or if the element is a foreign element, then there may be a single "/" (U+002F) character.

So both versions are totally valid but the `/` is just an optional character that hold no meaning.

Reply:
@jvasseur yes, it's valid. But the "cool kids" (W3C, MDN) no longer use it. Symfony Flex and recipes are like a new and refreshing Symfony, so I thought we should go for the cool new way of writing those empty elements 😎

Reply:
Yes, I'm for removing it to.

I just wanted to point to the part of the spec that was mentioning the fact that it's optional and doesn't hold any meaning.

Reply:
It holds a meaning: `<meta>` is HTML whereas `<meta />` is XML. That's the slight difference between the two.
Before, we used to write `<meta />` when writing xHTML 1.1, and `<META>` when writing HTML 4.
HTML 5 allows both syntaxes, but as said, "cool kids" write cool code 😎